pkgpanda/util.py

```diff
@@ -63,6 +65,20 @@ def variant_suffix(variant, delim='.'):
     return delim + variant
 
 
+def get_requests_retry_session(max_retries=4, backoff_factor=1, status_forcelist=[500, 502, 504]):
```

Comment:
Let's avoid using mutable objects like lists as parameter defaults, since that can lead to surprising behavior. Someone might try to mutate it without realizing it belongs to the module's scope, rather than the function's. We'd want to do something like this instead:

```
def get_requests_retry_session(..., status_forcelist=None):
    status_forcelist = status_forcelist or [500, 502, 504]
    ...
```
This way any mutations of `status_forcelist` won't impact other calls to `get_requests_retry_session()`.